1.js

```diff
@@ -0,0 +1,28 @@
+/*
+1. Написать функцию, которая принимает массив числовых значений, 
+и возвращает минимальное и максимальное значение массива
+*/
+
+"use strict"
+
+let arr = [10, 2, 3, 15, 46, 98];
+
+function maxNum(array){
+  if(!(array instanceof Array)){
```

Comment:
Separate statements with the line breaks

Reply:
Done.